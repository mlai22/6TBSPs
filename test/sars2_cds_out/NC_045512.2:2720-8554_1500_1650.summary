NC_045512.2:2720-8554_1500_1650	YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	259	1.494263593107011e-31
NC_045512.2:2720-8554_1500_1650	YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	259	1.494263593107011e-31
NC_045512.2:2720-8554_1500_1650	YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	259	1.494263593107011e-31
NC_045512.2:2720-8554_1500_1650	YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	259	1.494263593107011e-31
NC_045512.2:2720-8554_1500_1650	YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	32	3.359192048771984
NC_045512.2:2720-8554_1500_1650	YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	26	23.013902351420764
NC_045512.2:2720-8554_1500_1650	YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	26	23.013902351420764
NC_045512.2:2720-8554_1500_1650	YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	26	23.013902351420764
NC_045512.2:2720-8554_1500_1650	YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	20	35.03750596048832
NC_045512.2:2720-8554_1500_1650	YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	20	35.03750596048832
NC_045512.2:2720-8554_1500_1650	YP_009724390.1:1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	24	43.7094568851775
NC_045512.2:2720-8554_1500_1650	YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	126.38731069577798
NC_045512.2:2720-8554_1500_1650	YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	16	126.38731069577798
NC_045512.2:2720-8554_1500_1650	YP_009725298.1 nsp2 [polyprotein_range=YP_009724389.1:181-818] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	15	174.17910111920432
NC_045512.2:2720-8554_1500_1650	YP_009725308.1 helicase [polyprotein_range=YP_009724389.1:5325-5925] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	15	174.17910111920432
NC_045512.2:2720-8554_1500_1650	YP_009742609.1 nsp2 [polyprotein_range=YP_009725295.1:181-818] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	15	174.17910111920432
NC_045512.2:2720-8554_1500_1650	YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	17	183.41755356722194
NC_045512.2:2720-8554_1500_1650	YP_009724390.1:1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	17	183.41755356722194
NC_045512.2:2720-8554_1500_1650	YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	17	183.41755356722194
NC_045512.2:2720-8554_1500_1650	YP_009724390.1:1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	17	764.2398065300914
NC_045512.2:2720-8554_1500_1650	YP_009724390.1:1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	13	771.8945496639751
NC_045512.2:2720-8554_1500_1650	YP_009725298.1 nsp2 [polyprotein_range=YP_009724389.1:181-818] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	14	1080.192433469381
NC_045512.2:2720-8554_1500_1650	YP_009742609.1 nsp2 [polyprotein_range=YP_009725295.1:181-818] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	14	1080.192433469381
NC_045512.2:2720-8554_1500_1650	YP_009725311.1 2'-O-ribose methyltransferase [polyprotein_range=YP_009724389.1:6799-7096] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	14	1080.192433469381
NC_045512.2:2720-8554_1500_1650	YP_009724394.1:1-61 ORF6 protein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	13	2756.766248799911
NC_045512.2:2720-8554_1500_1650	YP_009725298.1 nsp2 [polyprotein_range=YP_009724389.1:181-818] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	13	2756.766248799911
NC_045512.2:2720-8554_1500_1650	YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	13	2756.766248799911
NC_045512.2:2720-8554_1500_1650	YP_009742609.1 nsp2 [polyprotein_range=YP_009725295.1:181-818] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]	13	2756.766248799911
