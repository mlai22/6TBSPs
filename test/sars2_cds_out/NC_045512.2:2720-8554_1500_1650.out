Frame: 1
Query: KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL
Length: 50
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 259
E-value: 1.494263593107011e-31

0     KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL  49  
      ||||||||||||||||||||||||||||||||||||||||||||||||||      
1318  KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL  1367


Frame: 1
Query: KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL
Length: 50
Subject: YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 4405

Score: 259
E-value: 1.494263593107011e-31

0     KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL  49  
      ||||||||||||||||||||||||||||||||||||||||||||||||||      
1318  KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL  1367


Frame: 1
Query: KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL
Length: 50
Subject: YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 259
E-value: 1.494263593107011e-31

0    KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL  49 
     ||||||||||||||||||||||||||||||||||||||||||||||||||     
500  KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL  549


Frame: 1
Query: KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL
Length: 50
Subject: YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 259
E-value: 1.494263593107011e-31

0    KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL  49 
     ||||||||||||||||||||||||||||||||||||||||||||||||||     
500  KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL  549


Frame: 3
Query: SANRQLYNHLPGSGFKWLHCRGGKDSA
Length: 27
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 32
E-value: 3.359192048771984

10    PGSGFKWLHCRGGKDS  25  
      ||**|**|*|**|**|      
3370  PGQTFSVLACYNGSPS  3385

10    PGSGFKWLHCRGGKDSA  26  
      ||**|**|*|**|**|*      
3370  PGQTFSVLACYNGSPSG  3386


Frame: 3
Query: SANRQLYNHLPGSGFKWLHCRGGKDSA
Length: 27
Subject: YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 4405

Score: 26
E-value: 23.013902351420764

12    SGFKW  16  
      |*|||      
2311  SSFKW  2315


Frame: 3
Query: SANRQLYNHLPGSGFKWLHCRGGKDSA
Length: 27
Subject: YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 26
E-value: 23.013902351420764

12    SGFKW  16  
      |*|||      
1493  SSFKW  1497


Frame: 3
Query: SANRQLYNHLPGSGFKWLHCRGGKDSA
Length: 27
Subject: YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 26
E-value: 23.013902351420764

12    SGFKW  16  
      |*|||      
1493  SSFKW  1497


Frame: -3
Query: EFLASH
Length: 6
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 20
E-value: 35.03750596048832

2    LASH  5  
     ||||     
915  LASH  918


Frame: -3
Query: EFLASH
Length: 6
Subject: YP_009725295.1:1-4405 ORF1a polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 4405

Score: 20
E-value: 35.03750596048832

2    LASH  5  
     ||||     
915  LASH  918


Frame: 3
Query: SANRQLYNHLPGSGFKWLHCRGGKDSA
Length: 27
Subject: YP_009724390.1:1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1273

Score: 24
E-value: 43.7094568851775

5    LYNHLPGSGFK  15 
     |||****|*||     
367  LYNSASFSTFK  377


Frame: -3
Query: EFLASH
Length: 6
Subject: YP_009725299.1 nsp3 [polyprotein_range=YP_009724389.1:819-2763] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 16
E-value: 126.38731069577798

3   ASH  5  
    |||     
98  ASH  100

2    LASH  5  
     ||*|     
291  LAKH  294


Frame: -3
Query: EFLASH
Length: 6
Subject: YP_009742610.1 nsp3 [polyprotein_range=YP_009725295.1:819-2763] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1945

Score: 16
E-value: 126.38731069577798

3   ASH  5  
    |||     
98  ASH  100

2    LASH  5  
     ||*|     
291  LAKH  294


Frame: -3
Query: EFLASH
Length: 6
Subject: YP_009725298.1 nsp2 [polyprotein_range=YP_009724389.1:181-818] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 638

Score: 15
E-value: 174.17910111920432

0    EFL  2  
     |||     
459  EFL  461


Frame: -3
Query: EFLASH
Length: 6
Subject: YP_009725308.1 helicase [polyprotein_range=YP_009724389.1:5325-5925] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 601

Score: 15
E-value: 174.17910111920432

0    EFL  2  
     |||     
497  EFL  499


Frame: -3
Query: EFLASH
Length: 6
Subject: YP_009742609.1 nsp2 [polyprotein_range=YP_009725295.1:181-818] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 638

Score: 15
E-value: 174.17910111920432

0    EFL  2  
     |||     
459  EFL  461


Frame: -1
Query: KNFLLLIRDNRW
Length: 12
Subject: YP_009724389.1:1-7096 ORF1ab polyprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 7096

Score: 17
E-value: 183.41755356722194

0     KNF  2   
      |||      
5171  KNF  5173


Frame: -1
Query: KNFLLLIRDNRW
Length: 12
Subject: YP_009724390.1:1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1273

Score: 17
E-value: 183.41755356722194

0     KNF  2   
      |||      
1072  KNF  1074


Frame: -1
Query: KNFLLLIRDNRW
Length: 12
Subject: YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 932

Score: 17
E-value: 183.41755356722194

0    KNF  2  
     |||     
779  KNF  781


Frame: 1
Query: KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL
Length: 50
Subject: YP_009724390.1:1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1273

Score: 17
E-value: 764.2398065300914

13   QGLNGYTV  20 
     |****||*     
689  QSIIAYTM  696

13   QGLNGYTVE  21 
     |****||**     
689  QSIIAYTMS  697


Frame: -2
Query: RISCFSLEIIDGRM
Length: 14
Subject: YP_009724390.1:1-1273 surface glycoprotein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 1273

Score: 13
E-value: 771.8945496639751

0    RIS  2  
     |||     
356  RIS  358


Frame: 3
Query: SANRQLYNHLPGSGFKWLHCRGGKDSA
Length: 27
Subject: YP_009725298.1 nsp2 [polyprotein_range=YP_009724389.1:181-818] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 638

Score: 14
E-value: 1080.192433469381

0    SAN  2  
     |||     
247  SAN  249


Frame: 3
Query: SANRQLYNHLPGSGFKWLHCRGGKDSA
Length: 27
Subject: YP_009742609.1 nsp2 [polyprotein_range=YP_009725295.1:181-818] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 638

Score: 14
E-value: 1080.192433469381

0    SAN  2  
     |||     
247  SAN  249


Frame: 3
Query: SANRQLYNHLPGSGFKWLHCRGGKDSA
Length: 27
Subject: YP_009725311.1 2'-O-ribose methyltransferase [polyprotein_range=YP_009724389.1:6799-7096] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 298

Score: 14
E-value: 1080.192433469381

24  DSA  26
    |||    
31  DSA  33


Frame: 1
Query: KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL
Length: 50
Subject: YP_009724394.1:1-61 ORF6 protein [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 61

Score: 13
E-value: 2756.766248799911

47  EIL  49
    |||    
12  EIL  14


Frame: 1
Query: KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL
Length: 50
Subject: YP_009725298.1 nsp2 [polyprotein_range=YP_009724389.1:181-818] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 638

Score: 13
E-value: 2756.766248799911

47   EIL  49 
     |||     
271  EIL  273


Frame: 1
Query: KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL
Length: 50
Subject: YP_009725307.1 RNA-dependent RNA polymerase [polyprotein_range=YP_009724389.1:4393-5324] [polyprotein=ORF1ab polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 932

Score: 13
E-value: 2756.766248799911

47   EIL  49 
     |||     
143  EIL  145


Frame: 1
Query: KVPTDNYITTYPGQGLNGYTVEEAKTVLKKCKSAFYILPSIISNEKQEIL
Length: 50
Subject: YP_009742609.1 nsp2 [polyprotein_range=YP_009725295.1:181-818] [polyprotein=ORF1a polyprotein] [organism=Severe acute respiratory syndrome coronavirus 2] [isolate=Wuhan-Hu-1]
Length: 638

Score: 13
E-value: 2756.766248799911

47   EIL  49 
     |||     
271  EIL  273


